README.md

```diff
@@ -44,19 +44,19 @@ This is in beta state and you can expect running into problems. If you run into
 
 ### Simple Flow
 
-[Publish your first datatoken](READMEs/datatokens_flow.md) - connect to Ethereum, create an Ocean instance, and publish.
```

Comment:
While codacy wants us to go to files with `-` instead of `_`, and therefore requiring `\` handler before every `-`. This makes the code _harder_ to read, the opposite of what codacy is supposed to achieve. 

Also note that python convention itself is for .py filenames to be snake case (include `_`) rather than camel case.

We should choose for ourselves, which we want:
1. Write a Codacy instruction to ignore the rule. Pro: better readability, still in line with py 
2. Follow this Codacy rule of `_` for .md files. Pros/cons: opposite of above

Thoughts?

Reply:
while python recommends snake case, .py files are not "linked" as an URL anywhere. These changes affect only markdown files, and are recommended by Codacy, but also here: https://github.com/bartvandebiezen/file-name-conventions/blob/master/README.md

see:
```
Never use spaces or underscores. Spaces are converted to %20 in URLs or can break an URL when shared. Underscores are difficult to see when the file name is displayed as an underlined link. Although the use of underscores does not impact your ranking that much, Google advices not to use underscores.
```

Also, the Google link from that readme: https://www.youtube.com/watch?v=AQcSFsQyct8

Markdown file names should conform to "linkable"/SEO-compliant etc. rules, not Python executable/source code file names.

Reply:
Ref. SEO -> if we want a search "get tokens OCEAN" to point to our github docs on getting ocean tokens, "get-tokens-OCEAN.md" is better than "get_tokens_OCEAN.md"

Reply:
Thanks very much for this info, these are great points! OK this is fine then.